web/src/AudioUpload.js

```diff
@@ -3,32 +3,70 @@ import { DropzoneArea } from 'material-ui-dropzone';
 import React from 'react';
 import Button from '@material-ui/core/Button';
 import foursquare from './APIClient';
-
 import FoursquareSuggest from './FoursquareSuggest';
+import { useHistory } from "react-router-dom";
 
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
 
-const handleUpload = (files) => {
+const handleUpload = (files, value) => {
   const file = files[0];
-  const ext = "mp3";
+  const ext = file.name.split('.').pop();
   const formData = new FormData();
   formData.append("ext", ext)
   formData.append("file", file)
+  if (value === "setJingle") {
+    formData.append("setJingle", 1)
+  }
+  else if (value === "setName") {
+    formData.append("setName", 1)
+  }
   foursquare.post('demo/marsbot/audio/upload', formData,
     {
       headers: {'Content-Type': 'multipart/form-data'}
-    }).then(console.log);
+    }).then(response => { 
+      console.log(response)
+    })
+    .catch(error => {
+        console.log(error.response)
+    });
 };
 
 export default function AudioUpload() {
   const [files, setFiles] = React.useState([]);
+  const [value, setValue] = React.useState([]);
+  const history = useHistory();
+  const submit = (files, value) => {
+    handleUpload(files, value);
+    history.push('/');
+  };
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
   return (
     <Container component="main" maxWidth="xs">
       <Box mt={8}>
-        <Typography component="h1" variant="h5">
+        <FormControl component="fieldset">
+          <FormLabel component="legend"> Primary action </FormLabel>
+          <RadioGroup defaultValue="disabled" name="customized-radios" value={value} onChange={handleChange} row>
+            <FormControlLabel value="setName" control={<Radio color="primary" />} label="Set name"/>
+            <FormControlLabel value="setJingle" control={<Radio color="primary" />} label="Set jingle"/>
+            <FormControlLabel
+              value="disabled"
+              disabled
+              control={<Radio color="primary" />}
+              label="(Disabled option)"
+            />
+          </RadioGroup>
+        </FormControl>
+
+        <Typography component="h6" variant= 'h6'>
```

Comment:
The `component` and `variant` here should be consistent with those of the upload Typography, should be `h1` and `h5` respectively.